zipkin2-api.yaml

```diff
@@ -425,6 +422,24 @@ definitions:
       tags:
         $ref: '#/definitions/Tags'
         description: 'Tags give your span context for search, viewing and analysis.'
+    example:
+      id: "352bff9a74ca9ad2"
+      traceId: "5af7183fb1d4cf5f"
+      parentId: "6b221d5bc9e6496c"
+      name: "query"
+      timestamp: 1461750040359000
+      duration: 5000
+      kind: "SERVER"
```

Comment:
sorry about the late feedback. this is not realistic as usually mysql is not instrumented.

here are some realistic spans you can choose one! https://github.com/openzipkin/zipkin/tree/master/zipkin-ui/testdata